### PR TITLE
Add Tezos support to AnySigner

### PR DESCRIPTION
**Pull request by leoneparise**
## Description
Add Tezos support to AnySigner.
Transaction sample is the same used in TezosSingerTests, but in JSON format:

```json
{
  "operationList": {
    "branch": "BL8euoCWqNCny9AR3AKjnpi38haYMxjei1ZqNHuXMn19JSQnoWp",
    "operations": [
      {
        "source": "tz1XVJ8bZUXs7r5NV8dHvuiBhzECvLRLR3jW",
        "fee": 1272,
        "counter": 30738,
        "gasLimit": 10100,
        "storageLimit": 257,
        "kind": 7,
        "revealOperationData": {
          "publicKey": "QpqYbIBypAofOj4qtaWBm7Gy+2mZPFAEg3gVudxVkj4="
        }
      },
      {
        "source": "tz1XVJ8bZUXs7r5NV8dHvuiBhzECvLRLR3jW",
        "fee": 1272,
        "counter": 30739,
        "gasLimit": 10100,
        "storageLimit": 257,
        "kind": 8,
        "transactionOperationData": {
          "destination": "tz1XVJ8bZUXs7r5NV8dHvuiBhzECvLRLR3jW",
          "amount": 1
        }
      }
    ]
  }
}
```

## Checklist

- [x] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

File: tests/Any/SignerTests.cpp [hunk 1]
```diff
@@ -59,8 +60,23 @@ TEST(Signer, EthereumTransactionSign) {
               output.output());
 }
 
+TEST(Signer, ETezosTransactionSign) {
```
Comment (hewigovens) [inline tests/Any/SignerTests.cpp]:
```suggestion
TEST(Signer, TezosTransactionSign) {
```

File: tests/Tezos/SignerTests.cpp [hunk 2]
```diff
@@ -67,6 +67,7 @@ TEST(TezosSigner, SignOperationList) {
     std::string expectedSignedBytes = expectedForgedBytesToSign + expectedSignature;
     auto key = PrivateKey(parse_hex("0x2e8905819b8723fe2c1d161860e5ee1830318dbf49a83bd451cfb8440c28bd6f"));
     auto signedBytes = Signer().signOperationList(key, op_list);
+    auto signetHex = hex(signedBytes.begin(), signedBytes.end());
```
Comment (hewigovens) [inline tests/Tezos/SignerTests.cpp]:
```suggestion
    auto signedHex = hex(signedBytes.begin(), signedBytes.end());
```

File: tests/Any/SignerTests.cpp [hunk 3]
```diff
@@ -15,7 +16,7 @@ using namespace TW;
 using namespace TW::Any;
 
 TEST(Signer, CosmosTransactionSign) {
-    auto transaction = "{\"accountNumber\":\"8733\",\"chainId\":\"cosmoshub-2\",\"fee\":{\"amounts\":[{\"denom\":\"uatom\",\"amount\":\"5000\"}],\"gas\":\"200000\"},\"memo\":\"Testing\",\"sendCoinsMessage\":{\"fromAddress\":\"cosmos1ufwv9ymhqaal6xz47n0jhzm2wf4empfqvjy575\",\"toAddress\":\"cosmos135qla4294zxarqhhgxsx0sw56yssa3z0f78pm0\",\"amounts\":[{\"denom\":\"uatom\",\"amount\":\"995000\"}]}}";
+    auto transaction = R"({"accountNumber":"8733","chainId":"cosmoshub-2","fee":{"amounts":[{"denom":"uatom","amount":"5000"}],"gas":"200000"},"memo":"Testing","sendCoinsMessage":{"fromAddress":"cosmos1ufwv9ymhqaal6xz47n0jhzm2wf4empfqvjy575","toAddress":"cosmos135qla4294zxarqhhgxsx0sw56yssa3z0f78pm0","amounts":[{"denom":"uatom","amount":"995000"}]}})";
```
Comment (hewigovens) [inline tests/Any/SignerTests.cpp]: 👍

**Comment by leoneparise** [comment]
Fixed all comments

**Comment by hewigovens** [comment]
Do we need an interface to tell dapps which chain we support?

**Comment by vikmeup** [comment]
> Do we need an interface to tell dapps which chain we support?

Currently you get call `trustProvider.getAccounts()` to get supported list of blockchains that exposed an address.

**Pull request merged**